.goreleaser.yml

```diff
@@ -13,6 +13,7 @@ builds:
       - 386
       - arm64
       - arm
+      - ppc64le
```

Comment:
Do we also need a docker image for power?

Reply:
I think we should, but I wanted to split this in iterations. First lets test binary build and then we can test the docker build.